docs/conf.py

```diff
@@ -3,13 +3,19 @@
 import datetime as dt
 import json
 import os
+import sys
+
+sys.path.insert(0, os.path.abspath("../src/uwtools"))
 
 with open("../recipe/meta.json", "r", encoding="utf-8") as f:
     _metadata = json.loads(f.read())
 
 copyright = str(dt.datetime.now().year)
+extensions = ["sphinx.ext.autodoc", "sphinx.ext.intersphinx"]
 html_logo = os.path.join("static", "ufs.png")
 html_theme = "sphinx_rtd_theme"
+intersphinx_mapping = {"python": ("https://docs.python.org/3", None)}
```

Comment:
See [these docs](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_mapping) if curious. This particular mapping lets Sphinx recognize references to built-in Python names and creates links from their appearance in our docs (e.g. in type annotations in function/method signatures) and link to the official Python docs for those names.